haystack/utils/hf.py

```diff
@@ -212,27 +212,6 @@ def resolve_hf_pipeline_kwargs(
     return huggingface_pipeline_kwargs
 
 
-def list_inference_deployed_models(headers: Optional[Dict] = None) -> List[str]:
```

Comment:
this function was only used in TGI generators and did not work as expected, restricting too much the available models